generated/nidcpower/nidcpower/session.py

```diff
@@ -4778,7 +4778,8 @@ def import_attribute_configuration_buffer(self, configuration):
         '''
         vi_ctype = _visatype.ViSession(self._vi)  # case S110
         size_ctype = _visatype.ViInt32(0 if configuration is None else len(configuration))  # case S160
-        configuration_ctype = get_ctypes_pointer_for_buffer(value=_converters.convert_to_bytes(configuration), library_type=_visatype.ViInt8)  # case B520
+        converted_configuration = _converters.convert_to_bytes(configuration)  # case B520
```

Comment:
I think it would be a slightly better name to call this `configuration_converted` so that you have

1. `configuration`
2. `configuration_converted`
3. `configuration_ctype`